src/doc-editors/ServerGroupsEditor.vue

```diff
@@ -51,25 +51,54 @@
                 </button>
               </p>
               <p class="control">
-                <button class="button is-small has-text-danger delete-document-button"
-                        title="Delete document"
-                        data-qa="delete-document"
-                        :class="{'is-loading': isDeleteLoading}"
-                        :disabled="selectedServerGroup?.id === '__default__'"
-                        @click="deleteDoc()">
-                  <span class="icon is-small">
-                    <i class="fas fa-trash"></i>
-                  </span>
-                  <span>
-                    Delete
+                  <span class="field has-addons">
+                    <span class="control">
+                      <button class="button is-small has-text-danger delete-server-group"
+                              data-qa="delete-server-group-btn"
+                              @click="toggleDeleteServerGroupDoc()">
+                        <span class="icon is-small">
+                          <i  class="fas fa-trash"></i>
+                        </span>
+                      </button>
+                    </span>
+                    <span class="control is-expanded confirm-delete"
+                          v-if="deleteServerGroupDoc">
+                      <input class="input  is-small width-200px delete-server-group-input"
```

Comment:
```suggestion
                      <input class="input is-small width-200px delete-server-group-input"
```
Removing extra whitespace between `input` and `is-small`